### ISSUE TITLE
Runtime error: error while loading shared libraries

### ISSUE THREAD

**Issue by smaret** [smaret/astrochem#77]
On Ubuntu Linux, the SUNDIALS library (installed by hand in `/usr/local` is correctly detected by the `configure` script, and `make` succeeds. However, most tests fail because of an error while loading the shared libraries.

```
% make check
(...)
make[2]: Entering directory '/astrochem/tests'
FAIL: input_test
```

The error log for `input_test` is the following:

```
tests/.libs/input_test: error while loading shared libraries: libsundials_nvecserial.so.2: cannot open shared object file: No such file or directory
FAIL input_test (exit status: 127)
```

This can be fixed by adding `/usr/local/lib` to the `LD_LIBRARY_PATH`, but `libtool` should in principle take care of these details.

**Comment by smaret** [comment]
Related issue: #47

**Comment by smaret** [comment]
Fixed with 50bd5c0.
